test/add-member.test.js

```diff
@@ -42,3 +43,75 @@ test('get added to a group', async (t) => {
     )
   )
 })
+
+test('add member', async (t) => {
+  const kaitiaki = Testbot()
+  const newPerson = Testbot()
+  kaitiaki.tribes2.start()
+  newPerson.tribes2.start()
+
+  try {
+    const group = await kaitiaki.tribes2.create()
+    t.true(group.id, 'creates group')
+
+    const authorIds = [newPerson.id, ssbKeys.generate().id]
+
+    let invite = await kaitiaki.tribes2.addMembers(group.id, authorIds, {
+      text: 'welcome friends',
+    })
+
+    invite = await p(kaitiaki.db.get)(invite.key)
```

Comment:
Pretty nasty to mutate the `let invite`, it doesn't hurt to make these things `const`.